Cargo.toml

```diff
@@ -132,6 +132,7 @@ sqlx = { version = "=0.8.1", path = ".", default-features = false }
 
 # Common type integrations shared by multiple driver crates.
 # These are optional unless enabled in a workspace crate.
+base64 = { version = "0.22.0", default-features = false, features = ["std"] }
```

Comment:
This should go down below `# Common utility crates` or just be added to `sqlx-core`'s dependencies directly.